consoles/sshVirtsh.pm

```diff
@@ -34,8 +34,8 @@ sub new {
     my $self = $class->SUPER::new($testapi_console, $args);
 
     $self->instance(get_var('VIRSH_INSTANCE', 1));
-    # default name
-    $self->name("openQA-SUT-" . $self->instance);
+    # default name is openQA-SUT-1
+    $self->name(get_var('VIRSH_GUESTNAME', "openQA-SUT-") . $self->instance);
```

Comment:
I just don't like VIRSH_GUESTNAME to have trailing separator.
But true, we don't have to use any separator and it's more future-proof not to hardcode anything. Disregard my comments.